.bazelrc

```diff
@@ -6,6 +6,20 @@ build --repository_cache=~/.cache/bazel/repos/
 build --features=external_include_paths
 build --generate_json_trace_profile --profile command.profile.gz
 
+# https://libcxx.llvm.org/Hardening.html
+build:opt --copt -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_FAST
+build:fastbuild --copt -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_EXTENSIVE
+build:dbg --copt -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_DEBUG
```

Comment:
should we add these in `hephaestus.bzl` copts there? so we get them directly also in forkify?

Reply:
I think this should be a global option in order to not get any ODR violations.